src/shared/ml-thirdparty-client/index.ts

```diff
@@ -157,4 +158,123 @@ export class Client {
 
     // TODO: Implement communication with Mojaloop.
   }
+
+  /**
+   * Gets a list of PISP/DFSP participants
+   */
+  public async getParticipants(): Promise<void> {
+    if (this.simulator) {
```

Comment:
Is this the standard for other Mojaloop code? This seems... a bit scary.

Reply:
Uhmm... this is only the internal implementation for the PISP demo server... Sorry if this looks a bit scary, but the reason is because the plugins to make request and the one used to simulate the callbacks from mojaloop are different (i.e., separated into `ml-thirdparty-client` and `ml-thirdparty-simulator` plugins). 

If you want to avoid having a variable like this, maybe it would be better to merge those plugins into one single plugin and make the variable private

Reply:
I think the best approach would be to make a `Client` interface, and 2 classes which implement it `SimulatedClient` and `LiveClient` (or a better name...)

That way, you can pass through either a `SimulatedClient` which does the simulated calls or the real client which actually connects to Mojaloop